### DISCUSSION TITLE
Netflix 人流下滑

### DISCUSSION THREAD

**Discussion by AngusGuei** [1101-GS3073-C/Python-Community-GS3073-C#405] [General]
Netflix 近年會下滑我不覺得是影劇的問題，以我個人來講就只是會在不同的影視平台來回使用，以致不會像以前一直泡在 Netflix 

https://technews.tw/2022/06/13/netflix-squid-game-returning-for-season-2/

**Comment by AngusGuei** [comment]
補充  https://3c.ltn.com.tw/news/49197

**Comment by lukiea** [comment]
他在不拍些好看的劇 我真的想退訂了，雖然星期三和艾蜜莉在巴黎還是很香

**Comment by Raywei0214** [comment]
，我們仍然需要對Netflix保持懷疑的態度。我們需要淘汰舊有的觀念，我們一般認為，抓住了問題的關鍵，其他一切則會迎刃而解。

**Comment by larry40305** [comment]
而這些並不是完全重要，更加重要的問題是，謹慎地來說，我們必須考慮到所有可能。我們不妨可以這樣來想: 所謂Netflix，關鍵是Netflix需要如何解讀。
